### ISSUE TITLE
Is there still interest in continuing this project?

### ISSUE THREAD

**Issue by ace-archer** [DeastinY/lazy-dma#2]
Hello! I Stumbled across this while looking for something of this functionality (which surprisingly there's very little that combines Hue + a soundboard of some sort, having Sonos functionality built in is even more awesome since that's also what I want to run sound off), and am super interested in contributing if there's some interest in this still. 

Thanks!

**Comment by DeastinY** [comment]
**tl;dr** Not right now.

Hey @ace-archer,
The focus of this project shifted a few times and rotated around integrating 3D-Audio placement for a 5.1 system, then transitioned to using AirPlay for audio and some more chaos left and right. This is also not even my first iteration 😅 
(check here: https://github.com/DeastinY/atmosphuer)
What I can still contribute to the discussion is that I've done more research in the past months on related work and completely agree. There's no (good) tool that combines this reasonably simply functionality well.
I've worked on this with https://github.com/Olfinator who has a repository for the UI, but I don't think he made it public.

We also recently stopped playing any kind of pen and paper, as such motivation is kinda at a low now.
If you want to continue working I could gladly transfer ownership of the repo and help out here and there. Beware though - iirc streaming to Sonos was a bitch (need to create a radio-stream or something)
